src/Elmish.WPF.Tests/DynamicViewModelTests.fs

```diff
@@ -1431,6 +1431,34 @@ module SubModelSelectedItem =
       test <@ vm.Dispatches = [ set (selectedSubModel |> ValueOption.map getId) m ] @>
     }
 
+  [<Fact>]
+  let ``when getting nonexistent selected item, should throw RuntimeBinderException with "TryGetMember FAILED: Failed to find an element" message`` () =
+    Property.check <| property {
+      let selectedItemName = "Foo"
+      let subModelSeqName = "Bar"
+      let bindings =
+        [ selectedItemName |> Binding.subModelSelectedItem (subModelSeqName, Some, ignore)
+          subModelSeqName |> Binding.subModelSeq ((fun _ -> []), ignore, (fun () -> [])) ]
+
+      let mutable error : string option = None
+      let loggingArgs =
+        { LoggingViewModelArgs.none
+          with
+            log =
+              { new Microsoft.Extensions.Logging.ILogger
+                with
+                  member _.BeginScope _ = { new IDisposable with member _.Dispose() = () }
+                  member _.IsEnabled _ = true
+                  member _.Log (_logLevel, _eventId, state, excn, formatter) =
+                    error <- formatter.Invoke(state, excn) |> Some } }
+      let viewModelArgs = ViewModelArgs.create 0.0 ignore "main" loggingArgs
+
+      let vm = DynamicViewModel(viewModelArgs, bindings)
+      raises<Microsoft.CSharp.RuntimeBinder.RuntimeBinderException> <@ vm.Get selectedItemName @>
+      let error = error
```

Comment:
This workaround isn't needed any more
https://github.com/dotnet/fsharp/issues/9291